src/frontend/komponenter/Fagsak/Behandlingsresultat/EndretUtbetalingAndelRad.tsx

```diff
@@ -33,6 +33,11 @@ const PersonCelle = styled.div`
     }
 `;
 
+const StyledButton = styled(Button)`
+    width: 6rem;
```

Comment:
Gikk for 6 rem fordi det gjør at teksten "Se mer" som kommer i lesevisning også blir pen